src/tests/banners/DigitalSubscriptionsBannerTest.ts

```diff
@@ -12,7 +12,7 @@ export const DigitalSubscriptionsBanner: BannerTest = {
     canRun: (targeting: BannerTargeting, pageTracking: BannerPageTracking) => {
         if (targeting.switches.remoteSubscriptionsBanner) {
             const region = readerRevenueRegionFromCountryCode(targeting.countryCode);
-            return region === 'united-kingdom' || region === 'united-states';
+            return !(region === 'australia' || region === 'rest-of-world');
```

Comment:
I've updated this as we want to show the DS banner to any region that isn't `australia ` or `rest-of-world`, for example `european-union`, who wouldn't see this with the current setup.